Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -316,7 +316,7 @@
 #endif
 #endif
 
-#if !defined(ENABLE_GPU_DRIVER_PREWARMING) && PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 101600
+#if !defined(ENABLE_GPU_DRIVER_PREWARMING) && PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 101600 && __MAC_OS_X_VERSION_MIN_REQUIRED < 130000
```

Comment:
Do we have a #define for blocking IOKit in WebContent? Or at least a #define for using the GPU Process for DOM rendering? Can we use that here instead?

Reply:
That is a good point! I believe we only have runtime settings for this, but I will check closer.

Thanks for reviewing!